NSS-Day-Cohort-42/rare-server-media-moguls
# Cb fix user email

#### Changes Made
1. removed unnecessary comma from get_users_by_mail in users/request.py
2. removed comments from users/request.py

​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-fix-user-email
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> should allow for get user by email in postman
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
